setup-db.js

```diff
@@ -0,0 +1,18 @@
+require('dotenv').config();
+
+var MongoClient = require('mongodb').MongoClient;
+
+MongoClient.connect(process.env.MONGO_URL, { useNewUrlParser: true })
+  .then(async (client) => {
+    let db = client.db();
+    
+    // Store no more than 400MB. One month's worth of exitnode logs
+    // is approx 21600 logs * 15KB/log = 324MB.
+    await db.createCollection('routeLog', { 'capped': true, size: 400000000 })
+    console.log('Created routeLog collection.');
+    
+    await db.collection('routeLog').createIndex({ timestamp: 1 });
+    console.log('Created routeLog collection index.');
+    
+    client.close();
+  }).catch((err) => console.error(err));
```

Comment:
IMO good not to fully swallow the error here. I'd encourage `.catch((error) => throw error)` (by default node.js will show the stack trace from that throw anyway).

But by not swallowing it, the exit code of running this script will indicate errors when they happen.

Reply:
:+1: